lib/matplotlib/mlab.py

```diff
@@ -2115,6 +2118,7 @@ def norm_flat(a, p=2):
         return np.sum(np.abs(a) ** p) ** (1 / p)
 
 
+@cbook.deprecated('2.1', alternative='np.linspace(xini, xfin, delta)')
```

Comment:
actually the alternative is wrong, but I don't think we really need to bother specifying one

Reply:
I think it would be worthwhile specifying 'np.linspace' as the alternative without including the arguments.  Or specify both: 'np.linspace or np.arange'.